src/client/Microsoft.Identity.Client/Platforms/netstandard13/NetStandardDeviceAuthManager.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http.Headers;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
+using Microsoft.Identity.Client.PlatformsCommon.Shared;
+using Microsoft.Identity.Client.OAuth2;
+
+namespace Microsoft.Identity.Client.Platforms.netstandard13
+{
+    class NetStandardDeviceAuthManager : IDeviceAuthManager
+    {
+        public bool TryCreateDeviceAuthChallengeResponseAsync(HttpResponseHeaders responseHeaders, Uri endpointUri, out string responseHeader)
```

Comment:
+1 to what @trwalke said w/the duplicate code

Reply:
Done.